packages/e2e-test-utils-playwright/src/editor/search-for-block.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Search for block in the global inserter
+ *
+ * @param {Editor} this
+ * @param {string} searchTerm The text to search the inserter for.
+ */
+
+export async function searchForBlock( this: Editor, searchTerm: string ) {
```

Comment:
TBH, I don't think we need this util and we can just inline it whenever we need it. In theory, we don't need to use this util that often anyway, we only need it if we're testing the global inserter.

For block tests (like in this PR), we can just use `pageUtils.insertBlock` to programmatically insert blocks. It would be easier, faster, and less flaky.

Reply:
I generally agree on inlining the utils if there are not used a lot. Fixing that now.